### PR TITLE
change default 5 to 1 as :scale in angle-vector

### PR DESCRIPTION
**Pull request by furushchev**
This change may affects on baxter interface.
So please change :scale of baxter's angle-vector to 5 @aginika 

ref: https://github.com/jsk-ros-pkg/jsk_pr2eus/issues/120

### CODE REVIEW

**Comment by k-okada** [comment]
this may affects to all robots, not only pr2. plese check @snozawa @garaemon @mmurooka 

another solution is to change only for pr2.

**Comment by furushchev** [comment]
I got approval of @garaemon .
Is there any serious problem with this change? > @snozawa @mmurooka

**Comment by garaemon** [comment]
LGTM because
* staro -> all the programs are old and it does not have scale option
* hrp2 -> min-time=1.0 and scale=1 is enough

**Comment by k-okada** [comment]
ok, so document should be updated too,
https://github.com/furushchev/jsk_pr2eus/blob/fast-default-scale/pr2eus/robot-interface.l#L252

also we need to confirm @orikuma, some day , staro will use latest
code, and jaxon will use latest code very soon. @YoheiKakiuchi
--
◉ Kei Okada

**Comment by k-okada** [comment]
original discussion ->
https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/86#issuecomment-69718776
--
◉ Kei Okada

**Comment by furushchev** [comment]
updated documentation.

**Comment by k-okada** [comment]
please be careful , everyone!

**Comment by snozawa** [comment]
How about to use `[msec]` for return value of `:angle-vector-duration` and `:min-time` argument instead of `[sec]`?

In all lines using `:angle-vector-duration`, 
return values are multiplied by 1000 and change from `[sec]` to `[msec]`.
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L258

Almost all Euslisp users are familiar with `[msec]` for duration (for example, `:angle-vector`), 
so using `[msec]` for duration is more simple and less confusing.

**Comment by garaemon** [comment]
> How about to use [msec] for return value of :angle-vector-duration and :min-time argument instead of [sec]?

Just a historical reason why :angle-vector-duration returns time in sec. Hope somebody update it.

File: pr2eus/robot-interface.l [hunk 1]
```diff
@@ -246,10 +246,10 @@
          (let ((max-time (apply #'max time-list)))
            (max max-time min-time))))))
   (:angle-vector
-   (av &optional (tm nil) (ctype controller-type) (start-time 0) &key (scale 5) (min-time 1.0))
+   (av &optional (tm nil) (ctype controller-type) (start-time 0) &key (scale 1) (min-time 1.0))
    "tm (time to goal in msec)
    if designated tm is faster than fastest speed, use fastest speed
-   if not specified, it will use 1/5 of the fastest speed .
+   if not specified, it will use 1/scale of the fastest speed .
    if :fastest is specefied use fastest speed calcurated from max speed"
```
Comment (k-okada) [inline pr2eus/robot-interface.l]: if not specifined, that is max speed, so we do not need :fastest anymore, is it correct?

https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/86#issuecomment-83123289

**Pull request merged**